RecoTracker/MkFitCore/src/MkFinder.cc

```diff
@@ -860,40 +874,44 @@ namespace mkfit {
     // are already done when computing chi2. Not sure it's worth caching them?)
 
     dprint("update parameters");
-    (*fnd_foos.m_update_param_foo)(Err[iP],
-                                   Par[iP],
-                                   Chg,
-                                   msErr,
-                                   msPar,
-                                   Err[iC],
-                                   Par[iC],
+    (*fnd_foos.m_update_param_foo)(m_Err[iP],
+                                   m_Par[iP],
+                                   m_Chg,
+                                   m_msErr,
+                                   m_msPar,
+                                   m_Err[iC],
+                                   m_Par[iC],
                                    N_proc,
                                    m_prop_config->finding_intra_layer_pflags,
                                    m_prop_config->finding_requires_propagation_to_hit_pos);
 
-    dprint("Par[iP](0,0,0)=" << Par[iP](0, 0, 0) << " Par[iC](0,0,0)=" << Par[iC](0, 0, 0));
+    dprint("m_Par[iP](0,0,0)=" << m_Par[iP](0, 0, 0) << " m_Par[iC](0,0,0)=" << m_Par[iC](0, 0, 0));
   }
 
   //=======================================================
   // isStripQCompatible : check if prop is consistent with the barrel/endcap strip length
   //=======================================================
-  bool isStripQCompatible(
-      int itrack, bool isBarrel, const MPlexLS &pErr, const MPlexLV &pPar, const MPlexHS &msErr, const MPlexHV &msPar) {
+  bool isStripQCompatible(int itrack,
+                          bool isBarrel,
+                          const MPlexLS &pErr,
+                          const MPlexLV &pPar,
+                          const MPlexHS &m_msErr,
```

Comment:
here the arguments should not have a prefix `m_` to avoid confusion with data members